### ISSUE TITLE
[Meta] Write unit tests

### ISSUE THREAD

**Issue by arschles** [deis/steward#68]
Unit tests for:
- [x] the `./mode/cf` package (#72)
  - This should probably come in the form of a generic unit test harness for each interface in `./mode/interfaces.go`, and then using it for the `./mode/cf` implementations
- [x] the `./k8s` package, where possible (#72)
- [x] the `./k8s/claim` package (#109, #111, #164)
- [x] the functions in `./mode/json_object.go` (#72)
- [x] ~~all the handlers in `./web/brokerapi` (#64)~~ (no longer relevant after #95)
